src/Symfony/Component/Security/Core/Encoder/NativePasswordEncoder.php

```diff
@@ -80,9 +80,13 @@ public function isPasswordValid(string $encoded, string $raw, ?string $salt): bo
             return false;
         }
 
-        if (0 === strpos($encoded, '$2')) {
-            // BCrypt encodes only the first 72 chars
-            return 72 >= \strlen($raw) && password_verify($raw, $encoded);
+        if (0 !== strpos($encoded, '$argon')) {
+            if (0 === strpos($encoded, '$2')) {
```

Comment:
why nesting this rather than keeping it above the `if (0 !== strpos($encoded, '$argon')) {` ? It increases the potential conflicts when merging branches up by indenting that code.